packages/rum-core/src/boot/preStartRum.spec.ts

```diff
@@ -37,6 +38,10 @@ const AUTO_CONFIGURATION = { ...DEFAULT_INIT_CONFIGURATION }
 const MANUAL_CONFIGURATION = { ...AUTO_CONFIGURATION, trackViewsManually: true }
 const FAKE_WORKER = {} as DeflateWorker
 const PUBLIC_API = {} as RumPublicApi
+const oldFetch = window.fetch
+const oldXHROpen = XMLHttpRequest.prototype.open
+const oldXHRSend = XMLHttpRequest.prototype.send
+const oldXHRAbort = XMLHttpRequest.prototype.abort
```

Comment:
**💬 suggestion:** ‏move this at the start of your test case.
Some other tests executing earlier could instrument those methods and not properly clean them up.